src/squeue.rs

```diff
@@ -240,14 +246,77 @@ impl SubmissionQueue<'_> {
         }
 
         for Entry(entry) in entries {
-            *self
+            *self.next_sqe() = *entry;
+            self.move_forward();
+        }
+
+        Ok(())
+    }
+
+    /// Attempts to push an opcode into the submission queue.
+    /// If the queue is full, an error is returned.
+    ///
+    /// # Safety
+    ///
+    /// Developers must ensure that parameters of the opcode (such as buffer) are valid and will
+    /// be valid for the entire duration of the operation, otherwise it may cause memory problems.
+    #[inline]
+    pub unsafe fn push_command<'a, T: PrepareSQE>(
```

Comment:
As a new interface, it should be unstable.

Reply:
Fixed